src/ng/rootScope.js

```diff
@@ -437,7 +437,7 @@ function $RootScopeProvider() {
        * If any one expression in the collection changes the `listener` is executed.
        *
        * - The items in the `watchExpressions` array are observed via standard $watch operation and are examined on every
-       *   call to $digest() to see if any items changes.
+       *   call to $digest() to see if any items change.
```

Comment:
This makes it sound as if the act of examining itself may cause the change. I think it is more clear to say: _"if any item(s) changed"_